src/docs/markdown/running.md

```diff
@@ -152,6 +156,23 @@ RestartSec=5s
 Then, save the file and exit the text editor, and restart the service for it to take effect:
 <pre><code class="cmd bash">sudo systemctl restart caddy</code></pre>
 
+### SELinux System Considerations
+
+On SELinux enabled systems you have two options:
+1. Compile Caddy and label the files yourself
+2. Install Caddy using the [COPR repo](https://copr.fedorainfracloud.org/coprs/g/caddy/caddy/), your systemd file and caddy binary will be created and labelled correctly. NB: if you want to recompile Caddy with modules, you will have to label the executable as described bellow.
```

Comment:
I suggest reordering these lines so we're recommending installing from COPR first.

Also:

- `NB:` is not necessary, just start the sentence with `If`
- `bellow` -> `below`.
- I'd write it "If you wish to use a custom build of Caddy, you'll need to..."

Also users may not always have compiled Caddy themselves, they might have used the `/download` page or grabbed a vanilla build from GitHub so we might want to use different wording to cover that too.

Reply:
I'll test the `/download` case tomorrow on a fedora machine and see how it behaves, since I didn't package it I can't speculate on how it works yet.

Thank you for the typo correction, I've updated them and will push tomorrow after I test the direct download case

Reply:
All the download page gives you is a Caddy binary with whatever plugins you choose. It's the same output as using xcaddy, in fact it's just a wrapper around xcaddy.